pkg/snapshot/snapshotter/snapshotter_test.go

```diff
@@ -902,6 +907,159 @@ var _ = Describe("Snapshotter", func() {
 				})
 			})
 		})
+
+		FDescribe("Scenarios to update full snapshot lease", func() {
```

Comment:
```suggestion
		Describe("Scenarios to update full snapshot lease", func() {
```

Reply:
This was also part of the testing commit. I will revert this too :)